src/code/templates/readWorkflowInput.m

```diff
@@ -0,0 +1,106 @@
+function [SimulationSet,TaskList,workflowType,dataFiles] = readWorkflowInput(workflowInputxls,sheet)
+% READWORKFLOWINPUT converts workflow xls to STructures
+%
+%  Inputs 
+%       workflowInputxls (string) name of xls file
+%       sheet (string) name of sheet, if empty first sheet is taken
+%  Outputs 
+%       SimulationSet (structure) stores property of MeanModelset or RunPoulationSet
+%       TaskList (structure) stores list of tasks
+%       workflowType (string)  type of workflow
+%       dataFiles (cellarray) strores name of data files
+%  
+
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+
+
+% read data
+[~,~,xlsdata] = xlsread(workflowInputxls,sheet);
```

Comment:
We should call xlsread in 'basic' mode. (then we will directly see if it will work under Linux)